src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -52,13 +56,26 @@ class TaskDeploymentService {
                 .containerName("guacamole")
                 .containerPort(containerPort)
                 .build()
-        val albRoutingRule = awsCommunicator.createAlbRoutingRule(correlationId, listener.listenerArn(),targetGroup.targetGroupArn(),"/$userName/*")
+        val albRoutingRule = awsCommunicator.createAlbRoutingRule(correlationId, listener.listenerArn(), targetGroup.targetGroupArn(), "/$userName/*")
 
         // IAM permissions
-        parsePolicyDocuments(additionalPermissions)
+        val accessPair = Pair("ACCESS_RESOURCES",
```

Comment:
This is a bit of a mess imo, some things I'd like to see
- I'm not sure what this is trying to achieve and where the permissions are coming from / why we need them. Are we able to split this out to a function where some docs can be added?
- You're using `listOf` and then `plus`, can we not create this list in a cohesive way?
- The function name `setArns` has no meaning and tells you nothing of what it does